### DISCUSSION TITLE
Add more webviews

### DISCUSSION THREAD

**Discussion by niutech** [WebView-CG/Compatibility-Data-Project#29] [Ideas]
Currently there are only 3 distinct webviews on caniwebview.com: iOS WKWebView, Android WebView and Edge WebView2. Could you please add more of them?
- Qt WebEngine
- WebKitGTK
- GeckoView
- Ultralight
- Servo

**Comment by NiklasMerz** [comment]
We already discussed adding more WebViews in our group and wold love to add important ones at some point. Right now we are aiming to included the WebViews that have or will get data from the BCD project.
